main_test.go

```diff
@@ -121,13 +121,17 @@ func TestCalcPgMappingsToUndoBackfill(t *testing.T) {
 
```

Comment:
cam you add a `source: true`, `target: true` test case as well?

Reply:
yup yup!